packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarks.js

```diff
@@ -103,6 +105,17 @@ function DrawingToolMarks({
             onDeselect={deselectMark}
           />
         )}
+        {isActive && mark.tool.type == 'freehandLine' && (
+          <LineControls
+            mark={mark}
+            scale={scale}
+            onDelete={deleteMark}
+            onDeselect={deselectMark}
+            dragStart={selectMark}
+            dragMove={moveMark}
+            dragEnd={endMoveMark}
```

Comment:
This is wiring up the control buttons to the drag-and-drop handlers for the active line, which then throws an error when the Delete button tries to run `selectMark` for a deleted line.
```suggestion
```

Reply:
Good catch - removed!